clients/lsp-elixir.el

```diff
@@ -100,6 +100,33 @@ finding the executable with `exec-path'."
   :group 'lsp-elixir
   :type 'file)
 
+(defcustom lsp-elixir-enable-test-lenses t
+  "Suggest Tests."
+  :type 'boolean
+  :group 'lsp-elixir
+  :package-version '(lsp-mode . "7.1"))
+
+(defun lsp-elixir--build-test-command (argument)
+  "Builds the test command from the ARGUMENT."
+  (let ((test-name (lsp-get argument :testName))
+        (module (lsp-get argument :module))
+        (describe (lsp-get argument :describe)))
+    (cond (module (concat "\"" "module:" module "\""))
+          ((not test-name) (concat "\"" "describe:" describe "\""))
+          (describe (concat "\"" "test:test " describe " " test-name "\"" ))
+          (t (concat "\"" "test:test " test-name "\"" )))))
+
+(lsp-defun lsp-elixir--run-test ((&Command :arguments?))
+  "Runs tests."
+  (let* ((argument (lsp-seq-first arguments?))
+         (file-path (lsp-get argument :filePath))
```

Comment:
is `filePath` a new member of `Command`?

Reply:
I have no clue, what do you mean? I am pretty new to elisp and lsp-mode, so please excuse my lack of knowledge.

Reply:
@kiennq `arguments?` is a list of `any`, so you can have anything in it.